docs/html/installation.md

```diff
@@ -81,8 +81,11 @@ pip is tested to work on the latest patch version of the Python interpreter,
 for each of the minor versions listed above. Previous patch versions are
 supported on a best effort approach.
 
-pip's maintainers do not provide support for users on older versions of Python,
-and these users should request for support from the relevant provider
+Other operating systems and Python versions are not supported by pip's
+maintainers.
+
+Users who are on unsupported platforms should request for support from their
+provider for the platform
```

Comment:
I don't think we can necessarily assume that people on unsupported platforms have a "provider" of pip. After all, it comes with Python and via `get-pip.py`, and neither of those routes require you to have a vendor.

How about

"""
Other operating systems and Python versions are not supported by pip's
maintainers.

Users who are on unsupported platforms should be aware that if they hit issues, they may have to resolve them for themselves. If they received pip from a source which provides support for their platform, they should request pip support from that source.
"""